### DISCUSSION TITLE
What does scale do in RandomResizedCrop?

### DISCUSSION THREAD

**Discussion by alejopaullier96** [albumentations-team/albumentations#1291] [Q&A] [answered]
Could you further explain what `scale` does in `RandomResizedCrop`?  As far as I understand from the brief parameter definition (_range of size of the origin size cropped_) it:
1. Randomly crops a portion of the image where the shape of the crop is random (height and width of the crop varies each time you execute the transformation) but restricted such that the area of the crop is between the defined bounds.
2. The tuple passed in `scale` defines the lower and upper bounds of the crop's area with respect to the original image. For example, `(0.1,0.9)` means the crop's area will be randomly between 10% and 90% of the original image's area.
3. Finally, after cropping is done, resize the crop to the desired area.

**Comment by Dipet** [comment, marked as answer]
`scale` -  Specifies the lower and upper bounds for the random area of the crop, before resizing. The scale is defined with respect to the area of the original image.

Yes, you correctly described how it is works. Implementaion is here: https://github.com/albumentations-team/albumentations/blob/master/albumentations/augmentations/crops/transforms.py#L351
